exporter/clients_test.go

```diff
@@ -1,16 +1,28 @@
 package exporter
 
 import (
+	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 func TestParseClientListString(t *testing.T) {
+	convertDurationToTimestampString := func(duration string) string {
```

Comment:
See above, If you make `durationFieldToTimestamp()` return a string then we won't need this function.

Reply:
returning string is not purpose of `durationFieldToTimestamp`. This would violate SRP.

Reply:
Not sure what SRP is.
But you always convert the int that's returned from `durationFieldToTimestamp()` to a string so instead of doing that in reach place I prefer you'd just have the function return a string and then the caller can use it right away.

Reply:
SRP = single-responsibility principle

(That's what I assume it means here.)

Maybe just rename durationFieldToTimestamp to durationFieldToTimestampString and make it responsible for the whole conversion? (It's still a single responsibility.)

Reply:
@zuiderkwast That would not make it comply with SRP, but I don't have enough time to convince everyone on this matter. I'll do it as soon as I get the time to do so. If anyone had some spare time to fix this before I do, go ahead.

Reply:
@zuiderkwast That would not make it comply with SRP, but I don't have enough time to convince everyone on this matter. I'll do it as soon as I get the time to do so. If anyone had some spare time to fix this before I do, go ahead.